Cargo.toml

```diff
@@ -101,10 +101,11 @@ string_cache = "0.7.3"
 flate2 = "1.0.6"
 structopt = "0.2.16"
 indexmap = {version = "1.0.2", features = ["serde-1"]}
+itertools = "0.8.0"
```

Comment:
I'm a little hesitant to bring in another dependency just for the one use of `batching` in the test helper. There's probably a simple-enough way to get the same result with just the normal iterator adapters.

Reply:
Replaced random sizes for maps in `random_structured_data` with fixed cycle.